README.md

```diff
@@ -93,13 +99,47 @@ In this task you import a repository from GitHub. This repository mostly consist
 
    b. `DBX_TOKEN` = `XXXXXX` (replace `XXXXXX` with a Databricks token)
    
-      You can create a Databricks token by going to the Databricks environment and under 'User Settings' create a new token
+      You can create a Databricks token by going to the Databricks workspace and under 'User Settings' create a new token
       ![User settings](media/dbx-usersettings.png "Go to user settings")
       Now you can generate a token by pressing the 'Generate New Token' button. The comment input is not relevant. In this case, "DBX_api" is chosen.
       ![Generate token](media/generate-token-1.png "Generate token")
       ![Copy token](media/generate-token-2.png "Copy token")
       Now copy this token to the variable `DBX_TOKEN` in the GitHub secret field.
 
+### Task 3: Set up Databricks Repos
+To support best practices for data science and engineering code development, Databricks Repos provides repository-level integration with Git providers. You can develop code in a Databricks notebook and sync it with a remote Git repository. Databricks Repos lets you use Git functionality such as cloning a remote repo, managing branches, pushing and pulling changes, and visually comparing differences upon commit.
+
+1. Create a GitHub personal access token following the [GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). After creating the token, make sure to authorize SSO.
+
+2. Set up the Git Integration in your Databricks workspace:
+
+   a. In your Databricks workspace, go to User Settings and select the Git integration tab:
+
+      ![Databricks User Settings](media/dbx-usersettings.png)
+   
+   b. Select 'GitHub' as Git provider, enter your GitHub username and enter the personal access token you just created:
+
+      ![Databricks Git Integration](media/databricks-git-integration.png)
+
+3. In your Databricks workspace, go to the **Repos** tab and add your personal repository (in which you have just imported the files for this workshop):
+
+   ![Setup Databricks Repos](media/databricks-repos-setup.png)
+
+### Task 4: Create Databricks cluster
+To be able to execute the notebooks that are now visible in Databricks Repos, a cluster is required as compute. 
+
+1. In your Databricks workspace, go to the **Compute** tab and select **Create new cluster**:
+
+   ![Databricks New Cluster](media/dbx-new-cluster.png)
+
+2. Create a cluster in standard cluster mode, with the latest Long Term Support (LTS) Databricks runtime version (10.4 LTS), and all other settings as default:
```

Comment:
Is this runtime ok or do we need the ML version?

Reply:
Updated to ML cluster